packages/wrangler/src/experimental-assets.ts

```diff
@@ -0,0 +1,15 @@
+import path from "node:path";
+import type { Config } from "./config";
+
+/**
+ * Returns the base path of the experimental assets to upload.
+ *
+ */
+export function getExperimentalAssetsBasePath(
+	config: Config,
+	experimentalAssetsDirectory: string | undefined
```

Comment:
NIT - it would be clearer if this was called something like `experimentalAssetsCommandLineArg`.

Reply:
addressed here: https://github.com/cloudflare/workers-sdk/pull/6346